packages/dialtone-tokens/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## [1.23.6-alpha.1](https://github.com/dialpad/dialtone/compare/dialtone-tokens/v1.23.5...dialtone-tokens/v1.23.6-alpha.1) (2024-01-22)
```

Comment:
hmm should alpha releases actually show up in this changelog?

Reply:
We were showing them before on dialtone-vue and legacy dialtone. I remember having issues with them but I think they were because we weren't merging changes back.

I'll check if there's a way to only put "production" changes on the changelog.